tests/unit/CurrencyUtilsTest.ts

```diff
@@ -123,8 +123,12 @@ describe('CurrencyUtils', () => {
             [2550, '25.50'],
             [25, '0.25'],
             [2500.5, '25.00'],
-        ])('Correctly converts %s to amount in units handled in frontend as a string', (amount, expectedResult) => {
-            expect(CurrencyUtils.convertToFrontendAmountAsString(amount)).toBe(expectedResult);
+            [null, ''],
+            [undefined, ''],
+            ['', '0.00'],
```

Comment:
we don't accept an empty string in the function as of now

Reply:
so should i just remove the empty string test?

Reply:
i removed it in the recent commit .. let me know what you think